javascript/fetch-repos.js

```diff
@@ -122,6 +122,15 @@
           }));
         }
       });
+
+      FourthWall.fetchDefer({
+        url: team.baseUrl + "/teams/" + teamId + "/members",
+        done: function (result) {
+          FourthWall.importantUsers = result.map(function(item) {
```

Comment:
Also, having it overwrite creates a race condition between this and `fetchReposFromGist`.